libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/FeatureConfigurationFlyout.tsx

```diff
@@ -181,6 +188,24 @@ export class FeatureConfigurationFlyout extends React.Component<
 
   private onRenderFooterContent = () => {
     const tooManyFeaturesSelected = this._selection.getSelectedCount() > 2;
+    const noChangesInSelectedFeatures =
+      // check that feature selection has not changed

```

Comment:
nit: why not just make these two separate variables then:

const featureSelectionChanged = ...
const numBinsChanged = ...
const selectedFeaturesChanged = featureSelectionChanged || numBinsChanged

Reply:
Sure, incorporated this (with slightly different naming)